_posts/20-08-05-PulsVPN dataset leaked.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Datadump met informatie over vermoedelijkgehackte PulseVPN systemen gelekt / Datadump with information on hacked PulseVPN systems leaked
+author: Frank Breedijk
+excerpt:  Een datadump met gevoelige informatie over meer dan 900 vermoedelijk gehackte PulseVPN system is gelekt. Het DIVD Meldpunt gaat Nederlandse slachtoffers informeren. / A datadump with sensitive information about more then 900 compromised PulseVPN systems has been leaked. DIVD Meldpunt is going to inform the Dutch victims.
+---
+<small>[English below](#english)</small>
+
+### Nederlands
+
+Een datadump met informatie over meer dan 900 warschijnlijk gecompromitteerde Pulse VPN systemen is publiek gemaakt. Deze dump bevat o.a. gebruikersnamen met wachtwoord hashes en plain tekst wachtwoorden. DIVD heeft een toegang tot deze dataset en heeft de taak op zich genomen om beheerders van gecompromitteerde Nederlandse IP adressen te waarschuwen. 
+
+ZDNet heeft een artikel gepubliceerd over deze dump:
+https://www.zdnet.com/article/hacker-leaks-passwords-for-900-enterprise-vpn-servers/
+
+Voor meer informatie zie: [case DIVD-2020-00009](https://securitymeldpunt.nl/cases/DIVD-2020-00009/)
+
+### English
+
+A datadumpt with information on more then 900 likely compromised PulseVPN systems has been made public. This dump contains usernames, password hashes and plain tekst passwords. DIVD has access to this dump and has taken it upon itself to inform the administrators of Dutch IP addresses in this dump.
+
+ZDNet has published an article on this dump: https://www.zdnet.com/article/hacker-leaks-passwords-for-900-enterprise-vpn-servers/
+
+More information at: https://securitymeldpunt.nl/cases/DIVD-2020-00009/
```

Comment:
Geen linkje in de engelse versie (naar de case)

Reply:
Fixed